src/models/backingImage.js

```diff
@@ -74,6 +74,25 @@ export default {
       })
       yield put({ type: 'query' })
     },
+    *startWS({
+      payload,
+    }, { select }) {
+      let ws = yield select(state => state.backingImage.ws)
+      if (ws) {
+        ws.open()
+      } else {
+        wsChanges(payload.dispatch, payload.type, '1s', payload.ns)
+      }
+    },
+    *stopWS({
+      // eslint-disable-next-line no-unused-vars
+      payload,
+    }, { select }) {
+      let ws = yield select(state => state.backingImage.ws)
+      if (ws) {
+        ws.close(1000)
```

Comment:
Should ws be reset by null/undefined?

Reply:
I feel there is no need to change ws to null/undefined, because if we already have ws, StartWs function will calling ws.open() directly.